remoteworkitem/scheduler_test.go

```diff
@@ -33,7 +33,21 @@ func TestMain(m *testing.M) {
 		&Tracker{},
 		&TrackerQuery{},
 		&TrackerItem{})
-	db.Model(&TrackerQuery{}).AddForeignKey("tracker_id", "trackers(id)", "RESTRICT", "RESTRICT")
+	db.Exec(`CREATE OR REPLACE FUNCTION create_constraint_if_not_exists (t_name TEXT, c_name TEXT, constraint_sql TEXT)
```

Comment:
Why not use migration.Perform here to avoid duplicate and guaranteed long-term mismatch between test DB and test setup?